nrf_rpc/nrf_rpc.c

```diff
@@ -379,6 +380,11 @@ static void internal_tx_handler(void)
 	case NRF_RPC_TASK_GROUP_INIT: {
 		const struct nrf_rpc_group *group = task.group_init.group;
 
+		if (task.group_init.signal_reboot) {
```

Comment:
Hmm...  Why should device send error instead of notifying application about rebooted remote and reply to remote?

Reply:
`nrf_rpc_err` does notify the application about the error, and only sends the error in certain cases. But I would say, you probably will like https://github.com/nrfconnect/sdk-nrfxlib/pull/1583 approach more than this one :)